src/styles/main.scss

```diff
@@ -1,3 +1,27 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+@import "./utils/reset";
+
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($min-block-size, 1fr));
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  &__block {
+    line-height: $min-block-size;
+    min-width: $min-block-size;
+    text-align: center;
```

Comment:
remember that this is grid, you can still use justify and align for proper centering